src/Heading.js

```diff
@@ -0,0 +1,42 @@
+import Text from './Text'
+import { fontSizes } from './theme'
+
+const Heading = Text.withComponent('h3')
+
+Heading.displayName = 'Heading'
+
+Heading.defaultProps = {
+  fontSize: fontSizes[4]
```

Comment:
My preference for heading in UI is to have `margin: 0` by default since margin collapse can be a confusing concept for some. Zeroing out margins could be achieved on a per element basis with `m={0}`, but I wonder if it'd be worth having that as the default for the component?

Reply:
Oh, also, since the `fontSize` prop is from styled-system, you can pass the index as the value: `fontSize: 4`

Reply:
@jxnblk , cool feedback. Updated the `Heading` component with following changes. Please review again at your convenience. Thanks.
- Zero out the margins with `m=0` in default props.
- Remove the unnecessary use `fontSizes` variable. Set default font sizes with primitive number such as `fontSize: 4`.